dd-sdk-android-compose/README.md

```diff
@@ -79,6 +79,19 @@ val configuration = Configuration.Builder(
     .build()
 ```
 
+#### Action tracking
+
+There is no automated instrumentation support for the action tracking in Jetpack Compose yet. However, you may report clicks using `trackClicks` method in the following manner:
+
+```kotlin
+Button(
+    onClick = trackClicks(targetName = "Open View") { ...click logic... }
```

Comment:
I think the added value of this is the abstraction, which may enable us to do internal improvements later, without changing the `RUMMonitor` API 👌.

Reply:
true, good point.